testsuite/MDAnalysisTests/lib/test_cutil.py

```diff
@@ -45,6 +52,10 @@ def test_unique_int_1d(values):
     assert type(res) == type(ref)
     assert res.dtype == ref.dtype
 
+def test_array_unique_int_1d():
+    universe=u()
```

Comment:
Instead of a universe, create numpy arrays with the appropriate dtypes so that you can test behavior _without_ needing a Universe.